### PR TITLE
[Improvement][docs] Unit test benchmark specification

### PR DESCRIPTION
**Pull request by CalvinKirs**
unit test benchmark specification

### CODE REVIEW

**Comment by gabrywu** [comment]
@CalvinKirs Let's invite @dailidong and @qiaozhanwei to review this PR, if there're no opposite suggestions, this can be merged

**Comment by CalvinKirs** [comment]
> @CalvinKirs Let's invite @dailidong and @qiaozhanwei to review this PR, if there're no opposite suggestions, this can be merged

Thank you very much for your review

File: docs/zh-cn/development/unit-test.md [hunk 1]
```diff
@@ -15,3 +15,94 @@ Unit Test编写参考[链接](https://github.com/apache/incubator-dolphinschedul
...
+### 4：轻量型
+
+即任何环境都可快速执行。
+
+这要求我们尽可能不要依赖太多组件，如各种spring bean之类的。在单元测试中，这些都是可被mock的，增加这些，会加大我们单测的执行速度，同时也可能会传递污染。
+
+对于一些数据库、其他外部组件等。尽可能也采用模拟客户端的形式，即不依赖于外部环境，（任何外部依赖的存在都会极大的限制测试用例的可迁移性和稳定性以及结果正确性），这同时也方便开发者在任何环境都能够进行测试。
+
+
+
```
Comment (yangyichao-mango) [inline docs/zh-cn/development/unit-test.md:43]:
```suggestion

```

File: docs/zh-cn/development/unit-test.md [hunk 2]
```diff
@@ -15,3 +15,94 @@ Unit Test编写参考[链接](https://github.com/apache/incubator-dolphinschedul
...
+### 2：自动性
+
+单元测试能够自动化进行。强制要求：所有的单元测试必须写在src/test下，同时方法命名应该符合规范。基准测试除外。
+
+### 3：可重复性
+
+多次执行（任何环境任何时间）结果唯一，且可以重复执行。
+
+
```
Comment (yangyichao-mango) [inline docs/zh-cn/development/unit-test.md:33]:
```suggestion

```

File: docs/zh-cn/development/unit-test.md [hunk 3]
```diff
@@ -15,3 +15,94 @@ Unit Test编写参考[链接](https://github.com/apache/incubator-dolphinschedul
...
+### 4：轻量型
+
+即任何环境都可快速执行。
+
+这要求我们尽可能不要依赖太多组件，如各种spring bean之类的。在单元测试中，这些都是可被mock的，增加这些，会加大我们单测的执行速度，同时也可能会传递污染。
```
Comment (yangyichao-mango) [inline docs/zh-cn/development/unit-test.md:38]:
```suggestion
这要求我们尽可能不要依赖太多组件，如各种 spring bean 之类的。在单元测试中，这些都是可被 mock 的，增加这些，会加大我们单测的执行速度，同时也可能会传递污染。
```

File: docs/zh-cn/development/unit-test.md [hunk 4]
```diff
@@ -15,3 +15,94 @@ Unit Test编写参考[链接](https://github.com/apache/incubator-dolphinschedul
   * 在现阶段，Delta更改代码的测试覆盖设定值为：>＝60%，越高越好。
   * 我们可以在这个页面中看到测试报告: https://codecov.io/gh/apache/dolphinscheduler
 
+## 单元测试基本准则
```
Comment (yangyichao-mango) [inline docs/zh-cn/development/unit-test.md:18]: Hi,
Please improve the doc according to the specifications[1].

[1] https://dolphinscheduler.apache.org/zh-cn/docs/development/document.html
image

**Comment by yangyichao-mango** [comment]
Please change the title to `[Improvement][docs] Unit test benchmark specification`. Thx a lot~.

File: docs/zh-cn/development/unit-test.md (hunk 4 above)
Comment (CalvinKirs) [reply docs/zh-cn/development/unit-test.md:18]: Thx.done

**Review by dailidong — APPROVED** [review]
+1
good job
if add more unit test examples will be better

**Pull request merged**